tests/chart_tests/test_nats_jetstream.py

```diff
@@ -145,3 +145,18 @@ def test_stan_with_jetstream_disabled(self, kube_version):
         )
 
         assert len(docs) == 4
+
+    def test_jetstream_hook_job_disabled(self, kube_version):
```

Comment:
We need tests with `"global": {"nats": {"jetStream": {"enabled": False}}}` that test the behavior of `createJetStreamJob`

Reply:
createJetStreamJob - the test case added validates these

Reply:
we are testing them as a part of TestNatsJetstream

Reply:
As long as we have coverage of all 4 of those possibilities then we're good.